netsurf 3.10-2: build fails on hrev55204 x86_64

This build compiled successfully on Haiku R1B2 x86_64. 
May fail on Haiku R1B3.

`COMPILE: frontends/beos/cookies.cpp
frontends/beos/cookies.cpp:41:10: fatal error: NetworkCookieJar.h: No such file or directory
 #include <NetworkCookieJar.h>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
Makefile:864: recipe for target 'build/beos-beos/frontends_beos_cookies.o' failed
make: *** [build/beos-beos/frontends_beos_cookies.o] Error 1
make: *** Waiting for unfinished jobs....
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2.
Error: Build has failed - stopping.`

~/haikuports/www-client/netsurf> uname -a
Haiku shredder 1 hrev55204 Jul  1 2021 06:03:31 x86_64 x86_64 Haiku

~/haikuports/www-client/netsurf> find /system/develop/headers -name NetworkCookieJar.h
/system/develop/headers/private/netservices/NetworkCookieJar.h

--- a/www-client/netsurf/netsurf-3.10.recipe
+++ b/www-client/netsurf/netsurf-3.10.recipe
@@ -8,14 +8,14 @@ features including tabbed browsing, bookmarks and page thumbnailing."
 HOMEPAGE="https://www.netsurf-browser.org/"
 COPYRIGHT="2003-2020 The NetSurf Browser project"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 # Do NOT use the netsurf-all package. It contains a bunch of subprojects which
 # we build as separate packages so they can easily be used by other projects.
 SOURCE_URI="https://download.netsurf-browser.org/netsurf/releases/source/netsurf-$portVersion-src.tar.gz"
 CHECKSUM_SHA256="36484429e193614685c2ff246f55bd0a6dddf31a018bee45e0d1f7c28851995e"
 PATCHES="netsurf-$portVersion.patchset"
 
-ARCHITECTURES="!x86_gcc2 x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86_64"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
 fi

--- a/www-client/netsurf/patches/netsurf-3.10.patchset
+++ b/www-client/netsurf/patches/netsurf-3.10.patchset
@@ -1,4 +1,4 @@
-From 2849315a8ea6860d890f8e70c5a45bcebd019f83 Mon Sep 17 00:00:00 2001
+From bf012faedb02f22c99eede1d3eb16840560cc193 Mon Sep 17 00:00:00 2001
 From: Adrien Destugues <pulkomandy@pulkomandy.tk>
 Date: Sat, 30 May 2020 11:24:15 +0200
 Subject: build fix for libutf8proc
@@ -18,5 +18,27 @@ index 628ef1f..f6e498a 100644
  int32_t idna_contexto[] = {
  	/* CONTEXTO codepoints which have a rule defined */
 -- 
-2.26.0
+2.30.2
+
+
+From 381383edd2a27f5cf118807bb917beb177139f3a Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Thu, 1 Jul 2021 09:01:05 +0100
+Subject: removed NetCookieJar.h from previous build
+
+
+diff --git a/frontends/beos/cookies.cpp b/frontends/beos/cookies.cpp
+index 91a9beb..c49775b 100644
+--- a/frontends/beos/cookies.cpp
++++ b/frontends/beos/cookies.cpp
+@@ -38,7 +38,6 @@ extern "C" {
+ #include <private/interface/ColumnListView.h>
+ #include <private/interface/ColumnTypes.h>
+ #include <GroupLayoutBuilder.h>
+-#include <NetworkCookieJar.h>
+ #include <OutlineListView.h>
+ #include <ScrollView.h>
+ #include <StringView.h>
+-- 
+2.30.2
 

